pkg/synchromanager/clustersynchro/default_resource_synchro.go

```diff
@@ -117,6 +122,16 @@ func (factory DefaultResourceSynchroFactory) NewResourceSynchro(cluster string,
 		synchro.metricsWriter = metricsstore.NewMetricsWriter(config.MetricsStore.MetricsStore)
 	}
 
+	if config.Event != nil {
+		synchro.eventSynchro = newEventSynchro(cluster, synchro,
+			informer.NewFilteredListerWatcher(config.Event.ListerWatcher, func(opts *metav1.ListOptions) {
+				// https://kubernetes.io/docs/concepts/overview/working-with-objects/field-selectors/#list-of-supported-fields
+				field := fields.Set{}
+				field["involvedObject.kind"] = config.Kind
```

Comment:
Despite the added filtering, renamed kind will still be incorrectly matched to the